src/lib/es2015.symbol.wellknown.d.ts

```diff
@@ -219,9 +219,9 @@ interface String {
     match(matcher: { [Symbol.match](string: string): RegExpMatchArray | null; }): RegExpMatchArray | null;
 
     /**
-     * Replaces first match with string or all matches with RegExp.
-     * @param searchValue A string or RegExp search value.
-     * @param replaceValue A string containing the text to replace for match.
+     * Passes a string and {@linkcode replaceValue} to the [Symbol.replace] method on {@linkcode searchValue}. This method is expected to implement its own replacement algorithm.

```

Comment:
```suggestion
     * Passes a string and {@linkcode replaceValue} to the `[Symbol.replace]` method on {@linkcode searchValue}. This method is expected to implement its own replacement algorithm.
```

(Suggestion, dunno if the stdlib has its own style rules about JSDoc. Also `@@replace` is technically the name for the method—it's only accessed via `Symbol.replace`)

Reply:
I like the formatting improvement, but I think `@@replace` is only useful for people who have read the spec.

Reply:
Yeah, I'm fine with `[Symbol.replace]`. FWIW, on MDN we consistently use `RegExp.prototype[@@replace]()` so I hope it's not _that_ obscure, but I can understand.